### ISSUE TITLE
Pipenv Install From Git Fails On Windows With Whitespace In Path

### ISSUE THREAD

**Issue by sco1** [pypa/pipenv#1260]
Attempting to install packages from a Git repository fails on my machine with am `It looks like a path. Does it exist ?` error. From the traceback it looks like there is an issue with the whitespace in my install path (possibly related to #228?). I have no issue with "regular" `pipenv install` calls to install from PyPi.

Short of moving/symlinking my repo directory, am I missing something obvious with my setup that's causing the issue?

##### Describe your environment

1. OS: `Windows 10 Pro x64 10.0.16299 N/A Build 16299`
1. Python version: `3.6.3`
1. Pipenv version: `pipenv, version 9.0.1`
1. Shell: Both cygwin64 + fish and cmd exhibit the same behavior

##### Steps to replicate
Project path is: `C:\Users\First Last\Documents\GitHub\Wumbot`  
(Cygwin: `cygdrive/c/users/First\ Last/Documents/GitHub/Wumbot/`)
```
pipenv install git+https://github.com/Rapptz/discord.py@rewrite#egg=discord.py[voice] --verbose
```

##### Actual result

```
First Last@Computer /c/c/u/F/D/G/Wumbot> pipenv install git+https://github.com/Rapptz/discord.py@rewrite#egg=discord.py[voice] --verbose
Installing git+https://github.com/Rapptz/discord.py@rewrite#egg=discord.py[voice]…
Installing 'git+https://github.com/Rapptz/discord.py@rewrite#egg=discord.py[voice]'
$ "C:\\users\\First Last\\Documents\\GitHub\\Wumbot\\.venv\\Scripts\\pip.exe" install  --src C:\users\First Last\Documents\GitHub\Wumbot\.venv\src --verbose   "git+https://github.com/Rapptz/discord.py@rewrite#egg=discord.py[voice]" -i https://pypi.python.org/simple --exists-action w
Warning: You installed a VCS dependency in non–editable mode. This will work fine, but sub-dependencies will not be resolved by $ pipenv lock.
  To enable this sub–dependency functionality, specify that this dependency is editable.
Config variable 'Py_DEBUG' is unset, Python ABI tag may be incorrect
Config variable 'WITH_PYMALLOC' is unset, Python ABI tag may be incorrect
Exception information:
Traceback (most recent call last):
  File "c:\users\first~1\docume~1\github\wumbot\venv~1\lib\site-packages\pip\_vendor\packaging\requirements.py", line 92, in __init__
    req = REQUIREMENT.parseString(requirement_string)
  File "c:\users\first~1\docume~1\github\wumbot\venv~1\lib\site-packages\pip\_vendor\pyparsing.py", line 1617, in parseString
    raise exc
  File "c:\users\first~1\docume~1\github\wumbot\venv~1\lib\site-packages\pip\_vendor\pyparsing.py", line 1607, in parseString
    loc, tokens = self._parse( instring, 0 )
  File "c:\users\first~1\docume~1\github\wumbot\venv~1\lib\site-packages\pip\_vendor\pyparsing.py", line 1379, in _parseNoCache
    loc,tokens = self.parseImpl( instring, preloc, doActions )
  File "c:\users\first~1\docume~1\github\wumbot\venv~1\lib\site-packages\pip\_vendor\pyparsing.py", line 3376, in parseImpl
    loc, exprtokens = e._parse( instring, loc, doActions )
  File "c:\users\first~1\docume~1\github\wumbot\venv~1\lib\site-packages\pip\_vendor\pyparsing.py", line 1383, in _parseNoCache
    loc,tokens = self.parseImpl( instring, preloc, doActions )
  File "c:\users\first~1\docume~1\github\wumbot\venv~1\lib\site-packages\pip\_vendor\pyparsing.py", line 3164, in parseImpl
    raise ParseException(instring, loc, self.errmsg, self)
pip._vendor.pyparsing.ParseException: Expected stringEnd (at char 6), (line:1, col:7)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\users\first~1\docume~1\github\wumbot\venv~1\lib\site-packages\pip\req\req_install.py", line 82, in __init__
    req = Requirement(req)
  File "c:\users\first~1\docume~1\github\wumbot\venv~1\lib\site-packages\pip\_vendor\packaging\requirements.py", line 96, in __init__
    requirement_string[e.loc:e.loc + 8]))
pip._vendor.packaging.requirements.InvalidRequirement: Invalid requirement, parse error at "'\\Documen'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\users\first~1\docume~1\github\wumbot\venv~1\lib\site-packages\pip\basecommand.py", line 215, in main
    status = self.run(options, args)
  File "c:\users\first~1\docume~1\github\wumbot\venv~1\lib\site-packages\pip\commands\install.py", line 312, in run
    wheel_cache
  File "c:\users\first~1\docume~1\github\wumbot\venv~1\lib\site-packages\pip\basecommand.py", line 276, in populate_requirement_set
    wheel_cache=wheel_cache
  File "c:\users\first~1\docume~1\github\wumbot\venv~1\lib\site-packages\pip\req\req_install.py", line 235, in from_line
    wheel_cache=wheel_cache, constraint=constraint)
  File "c:\users\first~1\docume~1\github\wumbot\venv~1\lib\site-packages\pip\req\req_install.py", line 91, in __init__
    "Invalid requirement: '%s'\n%s" % (req, add_msg))
pip.exceptions.InstallationError: Invalid requirement: 'Last\Documents\GitHub\Wumbot\.venv\src'
It looks like a path. Does it exist ?

Error:  An error occurred while installing git+https://github.com/Rapptz/discord.py@rewrite#egg=discord.py[voice]!
Invalid requirement: 'Last\Documents\GitHub\Wumbot\.venv\src'
It looks like a path. Does it exist ?
```

**Comment by kennethreitz** [comment]
closing for now, as wontfix. will come up again if it's a real issue.

**Comment by sco1** [comment]
Any suggestions on where else to go for a solution to this not actual issue?

**Comment by techalchemy** [comment]
I’m actually fairly confident this is fixed in master

**Comment by sco1** [comment]
Seems fixed in `9.1.0`
